frame/dutch-auction/src/mock/runtime.rs

```diff
@@ -154,6 +148,7 @@ impl pallet_assets::Config for Runtime {
 	type WeightInfo = ();
 	type AdminOrigin = EnsureSignedBy<RootAccount, AccountId>;
 	type GovernanceRegistry = GovernanceRegistry;
+	type ValidCurrency = ValidateCurrencyId;
```

Comment:
Perhaps rename this to `CurrencyValidator` or some other verb, to emphasize it's an action